SimpleWAWS/Controllers/ResourceController.cs

```diff
@@ -51,7 +52,15 @@ public Task<HttpResponseMessage> DropAndReloadFromAzure()
         }
 
         [HttpGet]
-        public Task<HttpResponseMessage> All()
+        public async Task<HttpResponseMessage> DeleteUserResource(string userIdentity)
```

Comment:
Yes. Just making it easier for admins to delete from browser, instead of crafting an API call. I can remove it if  needed :)